### DISCUSSION TITLE
How many laps do cars race in the Australian Formula One Grand Prix?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#714] [Galaxy Trivia] [answered]
**Bonus:** Who set the lap record time there?

**Comment by SANKARAMDAS** [comment]
Formula 1 Heineken Australian Grand Prix 2022, which takes place over 58 laps of the 5.303-kilometre Melbourne Grand Prix Circuit in Albert Park on Sunday, April 10.
Their drivers have amassed 43 wins, 35 pole positions and 130 podium finishes, along with four world titles (three for Jack Brabham and one for Alan Jones).

**Comment by thuongtruong1009** [comment]
As of my knowledge cutoff in September 2021, the Australian Formula One Grand Prix traditionally consisted of 58 laps around the Melbourne Grand Prix Circuit. However, please note that Formula One race details can change from year to year, including the number of laps, so it's advisable to check for the most up-to-date information regarding a specific race.

Regarding the lap record time, as of September 2021, the lap record at the Melbourne Grand Prix Circuit was set by Michael Schumacher of Germany during the 2004 Australian Grand Prix. He achieved a time of 1:24.125 driving for Ferrari. Again, it's worth noting that lap records can be broken or updated in subsequent years, so it's recommended to verify the current lap record for the Australian Grand Prix.

**Comment by peterbaccaro** [comment, marked as answer]
The Australian Formula One Grand Prix which is at the Melbourne Grand Prix Circuit (Albert Park) has a race distance of **58 laps** (306.124 km).

The best lap record with a time of 1:20.260 was set by **Charles Leclerc** (Ferrari) in 2022.
